source/blog/2018-06-22-the-ember-times-issue-52.md

```diff
@@ -13,7 +13,36 @@ Read either on the [Ember blog](https://www.emberjs.com/blog/2018/06/22/the-embe
 
 ---
 
-## [SECTION TITLE](section url)
+## [Native Class Constructor Update 🛠](https://github.com/emberjs/rfcs/pull/337)
+
+There is currently an open RFC proposing to change the behavior of EmberObject's constructor.
```

Comment:
We aimed to put anything that describes Ember objects, methods or similar things into code quotes (\`\`) in the past - I think it would make sense to put any mentions of EmberObject into ``, too  so, it'd become  `EmberObject`. 
What do you think?